src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figures implements Area, Draw {

```

Comment:
if you implement AreaCalculator and Drawable to Figure class you don't have to implement it in every separate figure class because they will inherit it

Reply:
yes i understand, inheritance